icicle-compiler/src/Icicle/Sea/IO/Zebra.hs

```diff
@@ -174,7 +174,6 @@ seaOfDefReadProgram state = vsep
   , "        /* run compute on the facts read so far */"
   , "        if (*fact_count != 0) {"
   , indent 12 $ vsep $ fmap (\i -> pretty (nameOfCompute i) <+> " (program);") computes
-  , "            *fact_count = 0;"
```

Comment:
other option is to modify psv_write_outputs to not call compute - but that sounds like it would require some scary changes to psv read